rcongui/src/components/PlayerInfo/PlayerInfo.js

```diff
@@ -304,8 +317,8 @@ const PlayerInfoFunc = ({ classes }) => {
                 </Grid>
                 <Grid item>
                   <Typography variant="h6">
-                    <Link href={makeSteamProfileUrl(steamId64)}>
-                      Steam Profile
+                    <Link href={makeSteamProfileUrl(steamId64, names[0]?.name)}>
```

Comment:
could you rename the function to something more general (not mentionning steam in the name anymore :))

Reply:
Thanks for calling out my laziness. :)